src/Repository/SVNRepository.php

```diff
@@ -137,7 +137,7 @@ public function hasProviders() {
 
 	//protected function configurePackageWithVendorTransportOptions(); //@TODO come back to this
 
-	public function whatProvides( Pool $pool, $name ) {
+	public function whatProvides( Pool $pool, $name, $byPassFilters = false ) {
```

Comment:
According to the error message it looks like the argument should be `$bypassFilters`!